cypress/e2e/components/header.cy.js

```diff
@@ -1,42 +1,37 @@
-import selectors from '../constants/selectors';
+import selectors, { componentSelectors } from '../constants/selectors';
```

Comment:
```suggestion
import { componentSelectors } from '../constants/selectors';
```
`selectors` existieren nicht mehr